bucket/vesktop.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.3.1",
+    "description": "A cross platform electron-based desktop app aiming to give you a snappier Discord experience with Vencord pre-installed",
+    "homepage": "https://vencord.dev/",
+    "license": "GPL-3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Vencord/Vesktop/releases/download/v0.3.1/Vesktop-Setup-0.3.1.exe#/dl.7z",
+            "hash": "2b777365eb5aacb0285ea8a4f8819938b669979db1f85fdd5eabb51c32988af4"
+        }
+    },
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Force -Recurse"
+        ]
+    },
+    "bin": "vesktop.exe",
+    "shortcuts": [
+        [
+            "vesktop.exe",
+            "Vencord Desktop"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/Vencord/Vesktop"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/Vencord/Vesktop/releases/download/v$version/Vesktop-Setup-$version.exe#/dl.7z"
+            }
+        }
+    }
+}
```

Comment:
Vesktop now has a portable folder, w/o a setup, available.
This means we can avoid having an installer script, which makes this easier to maintain.

Here is my suggested manifest:
```suggestion
{
    "version":  "0.3.3",
    "description": "A cross platform electron-based desktop app aiming to give you a snappier Discord experience with Vencord pre-installed",
    "homepage":  "https://github.com/Vencord/Vesktop",
    "license":  "GPL-3.0",
    "architecture": {
        "64bit": {
            "url": "https://github.com/Vencord/Vesktop/releases/download/v0.3.3/Vesktop-0.3.3-win.zip",
            "hash": "faca0728ee67c31469f532412c30c5c21aa3aa41283111a62c210313acb5b2c3"
        }
    },
    "bin": "Vesktop.exe",
    "shortcuts":  [[
        "Vesktop.exe",
        "Vesktop"
    ]],
    "checkver": {
        "github": "https://github.com/Vencord/Vesktop"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/Vencord/Vesktop/releases/download/v$version/Vesktop-$version-win.zip"
            }
        }
    }
}`

PS: It appears that your manifest downloads the archive as `dl.7z`, but you refer to it as `app-dl.7z` in your install script.
I am new to scoop manifests, but this looks like a mistake?

Reply:
PS: It appears that your manifest downloads the archive as `dl.7z`, but you refer to it as `app-dl.7z` in your install script.
I am new to scoop manifests, but this looks like a mistake?

Reply:
I am new to scoop manifests myself, your suggestion looks good though and I agree that a portable zip works better for scoop. Though I would keep the name "Vencord Desktop" as shortcut as that is the older name of the app. Unless that causes some other issues I'm not aware of..